rtos/mbed_lib.json

```diff
@@ -3,5 +3,22 @@
     "config": {
         "present": 1
     },
-    "macros": ["_RTE_"]
+    "macros": ["_RTE_"],
+    "target_overrides": {
```

Comment:
In `targets/targets.json` file `boot-stack-size` is set to 4kB and here is modified and set to 1kB with some exceptions (like for NRF boards). 
Is this done to distinguish stack size for builds with and without RTOS (4kB for non RTOS and  typically 1kB for builds with RTOS)?

Reply:
Yep, by default stack sizes  are 4K and they are overridden to 1K when the RTOS is present. Targets with special requirements on stack size, such as the nrf51, can further override for both the RTOS and non-RTOS configuration.